modules/dokkatoo-plugin/src/main/kotlin/formats/DokkatooFormatPlugin.kt

```diff
@@ -155,11 +155,9 @@ abstract class DokkatooFormatPlugin(
         version.map { v -> create("$this:$v") }
 
       with(dokkatooExtension.versions) {
-        dokkaPlugin(dokka("dokka-analysis"))
+        dokkaPlugin(dokka("analysis-kotlin-descriptors"))
         dokkaPlugin(dokka("templating-plugin"))
         dokkaPlugin(dokka("dokka-base"))
-        dokkaPlugin(dokka("kotlin-analysis-intellij"))
-        dokkaPlugin(dokka("kotlin-analysis-compiler"))
```

Comment:
Should the analysis dependencies prior to Dokka 1.9.0 be still kept if a consumer of Dokkatoo were to say use an older version of Dokka? Since this dependency appears to [only exist for Dokka 1.9.0+](https://central.sonatype.com/artifact/org.jetbrains.dokka/analysis-kotlin-descriptors/1.9.0/versions)

Reply:
No, get rid of the discontinued dependencies. If someone wants to use Dokkatoo + Dokka 1.8 they can manuall add the discontinued dependencies.